src/customFileParser/unpackGeometry/main.ts

```diff
@@ -102,10 +107,46 @@ export function unpackPrimitives(
   treeIndexNodeIdMap: number[],
   colorMap: THREE.Color[]) {
 
-  for (const sector of rootSector!.traverseSectors()) {
+  const groupSpaceRequirementsPerSector: any = {};
+  for (const sector of rootSector.traverseSectors()) {
+    const renderedPrimitiveCount: any = {};
```

Comment:
I don't understand the names here. You have both `renderedPrimitiveCount` as a map, but also `renderedPrimitive.count`.. What is `filePrimitive.count`? I think some renaming / comments would be good.

Reply:
Fixed